index.html

```diff
@@ -1,12 +1,146 @@
+        <a href="about.html"><span class="fa fa-info"></span> About</a>
+        <a href="#"><span class="fa fa-rocket"></span>Explore</a>
+      </div>
+    </nav>
+    <section class="hero-section">
+      <div class="welcome">
+        Hello, Great Minds!
+        <h1>ACCESS COURSES AT EASE</h1>
+      </div>
+      <div class="intro">
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Doloribus
+        possimus impedit illo facilis reprehenderit unde consequatur
+        perspiciatis fugit amet aliquid maxime facere quis pariatur est ea, aut
+        rerum ipsum blanditiis!
+      </div>
+    </section>
+    <section class="program">
+      <h3>
+        Main Program<br />
+        <hr />
+      </h3>
+      <ul>
+        <li>
+          <div>
+            <span class="fa fa-laptop-code icon"></span>
+            <span class="program-head">Lecture</span>
+          </div>
+          <div>Courses are simplified and flexible for "on-the-go" study.</div>
+        </li>
+
+        <li>
+          <div>
+            <span class="fa fa-archive icon"></span>
+            <span class="program-head">Exhibition</span>
+          </div>
+          <div>Explore some of the works of our students</div>
+        </li>
+
+        <li>
+          <div>
+            <span class="fa fa-comments icon"></span>
+            <span class="program-head">Forum</span>
+          </div>
+          <div>
+            Tune in on our weekly discussion and monthly general assesmbly
+          </div>
+        </li>
+
+        <li>
+          <div>
+            <span class="fa fa-chalkboard icon"></span>
+            <span class="program-head">Workshop</span>
+          </div>
+          <div>
+            Carefully tailored workshop with schedule flexibility and maximum
+            output,
+          </div>
+        </li>
+      </ul>
+      <button>Learn Now</button>
+    </section>
+    <section class="collaborators">
+      <h2>
+        Collaborators<br />
+        <hr />
+      </h2>
+      <ul>
+        <li class="modal">
+          <img src="./Assets/all3d_logo.jpg" alt="All 3D logo" />
+          <div>
+            <h3>All 3D</h3>
+            <p>
+              The best YT channel to access step by step tutorial on 3D
+              modelling and design. Theirawesome course outline helped achieve
+              the best results for our students.
+            </p>
+          </div>
+        </li>
+        <li class="modal">
+          <div>
+            <img src="./Assets/alison_logo.jpg" alt="Alison Logo" />
+          </div>
+          <div>
+            <h3>Alison</h3>
+            <p>
+              A renowed provider of the world's best free courses. Thus, they
+              really came in handy with course planning and layoutting
+            </p>
+          </div>
+        </li>
+      </ul>
+      <button>See More <span class="fa fa-arrow-down"></span></button>
+    </section>
```

Comment:
- [ ] In this section: it's supposed to generate speakers. I see that you generated collaborators instead. That was a great job 👏. But following the project requirements, you should create them dynamically using JavaScript. Please check the screenshot below:
<img width="721" alt="image" src="https://user-images.githubusercontent.com/77190203/152518908-82e86976-6801-4880-b9f3-9cb399cc3d20.png">